src/status_im2/common/keychain/events.cljs

```diff
@@ -81,31 +81,77 @@
          (str key-uid "-auth")
          #(callback (if % (oops/oget % "password") auth-method-none)))
         (callback nil))))))
-
 (defn save-user-password!
   [key-uid password]
   (keychain/save-credentials key-uid key-uid (security/safe-unmask-data password) #()))
 
+(defn get-user-password!
+  [key-uid callback]
+  (keychain/get-credentials key-uid
+                            #(if %
+                               (callback (security/mask-data (oops/oget % "password")))
```

Comment:
threading maybe makes it a bit more readable:
```
(-> (oops/get % password)
     (security/mask-data)
     (callback))
```
or 
```
(as-> % $
   (oops/oget $ "password")
   (security/mask-data $)
   (callback $))
```

also, you can have the `if` inside, since you are calling callback regardless:

```
(callback (when %
                  (-> (oops/get...))
```

maybe there's also better ways, but it's all fine to be honest

Reply:
went with the last option